CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.21)
 
-set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD 20)
```

Comment:
I don't know if MinGW is up to date with this, atleast not for std (I have the latest, and the latest std is v17). So not sure if this is just MinGW being outdated (although it seems to compile, when I look at the GH action).

Is there an std v20?

Reply:
Interesting... I have:
`g++.exe (Rev5, Built by MSYS2 project) 11.2.0`
it supports `-std=c++20`. According to the GCC documentation, that flag exists in GCC version 10 and higher.
Do you use MSYS2 as well?

Reply:
(I use `git bash` btw, I run this on a windows machine). Not sure about MSYS2 btw. I just install MinGW. Not sure if that uses MSYS in anyway.